### DISCUSSION TITLE
Production build is not working

### DISCUSSION THREAD

**Discussion by aquibbaig** [gatsbyjs/gatsby#30996] [Help] [answered]
## Description
Hi, I just pulled a starter template and deployed it in `surge` only to find that it shows a blank page. To double-check I ran `gatsby build` and opened up the `index.html` file in `public/` directory. I found that resources are not getting loaded properly. Also opened an issue at https://github.com/JaeYeopHan/gatsby-starter-bee/issues/257 but found that this happens with every starter library I pull.

Describe the issue that you're seeing.
![Screenshot](https://user-images.githubusercontent.com/26324376/115674450-a73a6a80-a31b-11eb-9301-bb0220bbc916.png)

### Steps to reproduce
- pull any starter template
- run `npm run build/gatsby build`
- open `index.html` in `public/` 

### Expected result
For starters, there should not be any errors on the console. I am not able to move to other pages or click on any stuff.

### Environment
```
  System:
    OS: Linux 4.18 Red Hat Enterprise Linux 8.0 (Ootpa)
    CPU: (8) x64 Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz
    Shell: 4.4.19 - /bin/bash
  Binaries:
    Node: 12.22.1 - ~/.nvm/versions/node/v12.22.1/bin/node
    Yarn: 1.22.5 - /usr/bin/yarn
    npm: 6.14.12 - ~/.nvm/versions/node/v12.22.1/bin/npm
  Languages:
    Python: 3.6.8 - /usr/bin/python
  Browsers:
    Chrome: 89.0.4389.90
    Firefox: 78.8.0esr
  npmPackages:
    gatsby: ^3.3.1 => 3.3.1
    gatsby-plugin-feed: ^3.2.0 => 3.2.0
    gatsby-plugin-gatsby-cloud: ^2.2.0 => 2.2.0
    gatsby-plugin-google-analytics: ^3.2.0 => 3.2.0
    gatsby-plugin-image: ^1.2.1 => 1.2.1
    gatsby-plugin-manifest: ^3.2.0 => 3.2.0
    gatsby-plugin-offline: ^4.2.0 => 4.2.0
    gatsby-plugin-react-helmet: ^4.2.0 => 4.2.0
    gatsby-plugin-sharp: ^3.2.1 => 3.2.1
    gatsby-remark-copy-linked-files: ^3.2.0 => 3.2.0
    gatsby-remark-images: ^4.2.0 => 4.2.0
    gatsby-remark-prismjs: ^4.2.0 => 4.2.0
    gatsby-remark-responsive-iframe: ^3.2.0 => 3.2.0
    gatsby-remark-smartypants: ^3.2.0 => 3.2.0
    gatsby-source-filesystem: ^3.2.0 => 3.2.0
    gatsby-transformer-remark: ^3.2.0 => 3.2.0
    gatsby-transformer-sharp: ^3.2.0 => 3.2.0
```

I think this is a problem with version mismatch but I am not sure.

**Comment by LekoArts** [comment, marked as answer]
You'll need to serve the build artifacts (`public`) with a webserver. Locally you can do this with `gatsby serve`
https://www.gatsbyjs.com/docs/reference/gatsby-cli/#serve

**Reply by aquibbaig** [reply]
@LekoArts What a relief! Thanks a lot, no errors now!
